classes/template/PKPTemplateManager.inc.php

```diff
@@ -190,6 +192,22 @@ function initialize() {
 				);
 			}
 
+			// Register color picker assets on the appearance page
+			$this->addJavaScript(
+				'spectrum',
+				Request::getBaseUrl() . '/lib/pkp/js/lib/jquery/plugins/spectrum/spectrum.js',
```

Comment:
`$this->_request` appears to be available here. Wherever possible, I'm trying to avoid static calls to `Request::` functions. These will eventually need cleanup.